docs/user-manual/xr/vr/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: VR
-sidebar_position: 4
+sidebar_position: 21
```

Comment:
Why is this number so high?

Reply:
So that we can fit other docs before it with manual position control without a need to bump it manually each time.